packages/contracts/test/dollar/core/StakingShare.t.sol

```diff
@@ -7,6 +7,12 @@ import {IMetaPool} from "../../../src/dollar/interfaces/IMetaPool.sol";
 import {StakingShare} from "../../../src/dollar/core/StakingShare.sol";
 import "../../../src/dollar/libraries/Constants.sol";
 
+contract StakingShareOneChange is StakingShare {
+    function hasUpgraded() public pure returns (bool) {
```

Comment:
can you refactor this to be a conditional?

Reply:
The call itself fails prior to the upgrade because the selector does not exist so there's no need but I will rework it

Reply:
you may add a test (or point me to if it's alrady added) and keep it this way too if it does the intended

Reply:
It's the bottom-most test in the same file